packages/@sanity/desk-tool/src/pane/DocumentsPaneMenu.js

```diff
@@ -8,7 +8,7 @@ import IconThumbnails from 'part:@sanity/base/th-large-icon'
 // import IconSettings from 'part:@sanity/base/cog-icon'
 import IconNew from 'part:@sanity/base/plus-circle-icon'
 
-const TEST_CARDS_AND_THUMBNAILS = __DEV__
+const TEST_CARDS_AND_THUMBNAILS = false
```

Comment:
Is this intentional?

Reply:
Yep. We dont want to expose this to the studios running on local host yet